OpenTrons/opentrons-api
# feat(protocol-designer,-step-generation): add skeleton for plate reader steps

# Overview

Here I add a skeleton step form for plate reader step addition (hidden behind feature flag). I also add the a new `lidOpen` property to the plate reader's temporal properties interface since we will need to know this state during step generation in order to perform initializations/readings and to move labware to and from the module.

Closes AUTH-1196

## Test Plan and Hands on Testing

- ensure absorbance reader feature flag is on
- create or import a Flex protocol
- edit protocol and find 'add step' on timeline toolbox
- verify that absorbance reader step option is present and dummy stepform toolbox opens

## Changelog

- add plate reader step skeleton
- update plate reader temporal properties

## Review requests

- see test plan

## Risk assessment

low
